integrations/kube-agent-updater/pkg/img/cosign.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package img
+
+import (
+	"context"
+	"crypto"
+	"encoding/hex"
+
+	"github.com/docker/distribution/reference"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/gravitational/trace"
+	"github.com/opencontainers/go-digest"
+	"github.com/sigstore/cosign/v2/pkg/cosign"
+	ociremote "github.com/sigstore/cosign/v2/pkg/oci/remote"
+	"github.com/sigstore/sigstore/pkg/cryptoutils"
+	"github.com/sigstore/sigstore/pkg/signature"
+)
+
+const hashAlgo = crypto.SHA256
```

Comment:
Can we add a comment to this const?
Some docs:
https://docs.docker.com/engine/reference/commandline/pull/
or
https://docs.digitalocean.com/glossary/digest/